OG/OG/AST/ASTNode.cs

```diff
@@ -1,5 +1,8 @@
 ﻿namespace OG.AST
 {
+    /// <summary>
+    /// The template for all nodes in the AST, in which all Nodes must inherit from.
+    /// </summary>
     public abstract class ASTNode
```

Comment:
Er måske ikke nødvendig, men ser det som en god fremtidssikring for at vi er sikre på, at alle Nodes er af samme type, samt giver muligheden for at samle implementation i denne klasse, hvis alle Nodes ender med at bruge det.